packages/storefront-lwc/src/modules/commerce/router/router.js

```diff
@@ -38,11 +40,27 @@ const router = new UniversalRouter(
 );
 
 const render = location => {
-    router.resolve(location).then(content => {
-        document.getElementById('content').innerHTML = content; // eslint-disable-line
```

Comment:
The main point of this change is to no longer use `innerHTML` with an interpolated string. Instead, now calling `document.createElement` and setting attributes should fix a lot of the problems.